deno.json

```diff
@@ -19,21 +19,27 @@
   },
   "compilerOptions": { "jsx": "react-jsx", "jsxImportSource": "preact" },
   "imports": {
+    "$fresh/": "https://deno.land/x/fresh@1.7.1/",
     "@/": "./",
-    "$fresh/": "https://raw.githubusercontent.com/denoland/fresh/60220dd33b5b0f6b5c72927c933dbc32a3c4734e/",
-    "preact": "https://esm.sh/preact@10.19.2",
-    "preact/": "https://esm.sh/preact@10.19.2/",
-    "preact-render-to-string": "https://esm.sh/*preact-render-to-string@6.2.2",
+    "@deno/gfm": "jsr:@deno/gfm@^0.9.0",
```

Comment:
using this version of gfm will remove the warning related to emoji using `assert` instead of `with` for type assertion.

Related: https://github.com/denosaurs/emoji/pull/15